test/map-spec.js

```diff
@@ -1,34 +1,39 @@
 import MapGL from '../src';
 
-import document from 'global/document';
-/* global process */
 import React from 'react';
 import ReactDOM from 'react-dom';
+import document from 'global/document';
 import test from 'tape-catch';
 
-/* eslint-disable no-process-env */
```

Comment:
OK. Admit it is a little cleaner code wise, but it is also nice to show explicitly what warnings we decide to ignore. Is there a one line syntax that allows us to disable a specific warning? Just curious, I am fine either way.